resources/idlharness.js

```diff
@@ -1551,6 +1795,35 @@ IdlInterface.prototype.add_iterable_members = function(member)
            { name: "thisValue", idlType: {idlType: "any"}, optional: true}]}));
 };
 
+IdlInterface.prototype.test_to_json_operation = function(memberHolderObject, member) {
+    if (member.is_to_json_regular_operation()) {
```

Comment:
Given the name of this method, I think it's reasonable to expect the specified member to be a `toJSON` operation. This condition make it a little too lax, since callers will receive no feedback if they provide an inappropriate object. It seems like this condition really belongs at the call site. Would you agree?